app-editors/diakonos/diakonos-0.9.7.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+USE_RUBY="ruby26 ruby27 ruby30"
+
+inherit ruby-ng
+
+DESCRIPTION="A Linux editor for the masses"
+HOMEPAGE="https://git.sr.ht/~pistos/diakonos"
+SRC_URI="https://github.com/Pistos/diakonos/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~riscv ~x86"
+IUSE="doc test"
+
+ruby_add_rdepend "dev-ruby/curses"
+
+ruby_add_bdepend "doc? ( dev-ruby/yard )
+	test? ( dev-ruby/bacon )"
+
+each_ruby_install() {
+	${RUBY} install.rb --dest-dir "${D}" --doc-dir /usr/share/doc/${P} || die "install failed"
```

Comment:
oh, and no ${PF}? Does the application expect it in ${P} or something?

(You'll end up getting QA warnings in portage when you bump to e.g. -r1 in future right now)